core/src/main/java/org/openedx/core/ui/ComposeCommon.kt

```diff
@@ -1136,33 +1137,40 @@ fun BackBtn(
 }
 
 @Composable
-fun ConnectionErrorView(
-    modifier: Modifier,
-    onReloadClick: () -> Unit,
+fun ConnectionErrorView(onReloadClick: () -> Unit) {
+    FullScreenErrorView(errorType = ErrorType.CONNECTION_ERROR, onReloadClick = onReloadClick)
+}
+
+@Composable
+fun FullScreenErrorView(
```

Comment:
it's not bad to have modifier as argument, we just can add default value to it: `modifier: Modifier = Modifier`
and use named arguments when call fun